src/cli/post-install.ts

```diff
@@ -1,53 +1,68 @@
-import { executeCommand } from "./utils.js";
+import { executeCommand, download } from "./utils.js";
 import signal from "signale";
+import os from "os";
+import fs from "fs";
 
 const { Signale } = signal;
 
 const signale = new Signale({ scope: "postinstall", interactive: true });
 
-async function download(url: string, verbose = false) {
-  await executeCommand(`curl -LOf ${url}`, verbose);
-}
+const PLATFORM = os.platform();
+const ARCH = os.arch();
+console.log(`Current platform: ${PLATFORM}, current architecture: ${ARCH}`);
 
-const OS = await executeCommand("uname -s", false);
-const ARCH = await executeCommand("uname -m", false);
+const SUPPORTED_PLATFORMS = ["linux", "darwin"]; // Unsaported platforms: 'win32', 'aix', 'freebsd', 'openbsd', 'sunos', 'android'
```

Comment:
Oh, changes below are made for windows support, but there's still some part of the build that doesn't support windows. Am I right?

Reply:
It's just part of the effort. So Windows is not on the list just yet.